apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -120,6 +121,26 @@ export class Utilities {
     return homeFolder;
   }
 
+  /**
+   * Returns true if rush is running on a CI.
+    *
+   * The package is-ci is to detect this.
+   * Set environment variable RUSH_CI_MODE to override:
+   *   To force CI mode, set the environment variable to CI
+   *   To force non-CI mode, set the environment variable to NON-CI
+   */
+  public static isCI(): boolean {
+    if (process.env.RUSH_CI_MODE === 'CI') {
```

Comment:
Expose this in `RushConfiguration` and put the env processing in [`EnvironmentConfiguration`](https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/src/api/EnvironmentConfiguration.ts)

Reply:
Sounds good